sklearn/decomposition/online_lda.py

```diff
@@ -138,6 +138,8 @@ class LatentDirichletAllocation(BaseEstimator, TransformerMixin):
 
     .. versionadded:: 0.17
 
+    Read more in the :ref:`User Guide <LatentDirichletAllocation>.
```

Comment:
For some unknown reason sphinx builts successfully but fails to update the docs on my machine(I tried this with other files as well), I tried deleting .doctree and making build using bare sphinx-build command as well but nothing seems to work, I can't figure out why.